src/main.c

```diff
@@ -302,7 +286,24 @@ main (int argc, char *argv[])
                               time_now, time_str_appended, current_waste_num);
 
             if (info_status == 0)
+            {
```

Comment:
Yeah, sorry about that, I'm pretty bad at git (never used it for colaborating before). So, what I did here (in my local repo) was the following:

- Merge local develop with upstream develop
- Revert the commit that shouldn't be there
- Merge issue_20 with develop, fixing the conflicts

Should this fix it? If so, I'll push it and do another pull request.